[2024-03-27 18:51:11,194] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: bigdag.clean_task a9b5d366-a1b9-4827-8e6a-0baba9a90cdc [queued]>
[2024-03-27 18:51:11,204] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: bigdag.clean_task a9b5d366-a1b9-4827-8e6a-0baba9a90cdc [queued]>
[2024-03-27 18:51:11,204] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-03-27 18:51:11,204] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2024-03-27 18:51:11,205] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-03-27 18:51:11,217] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): clean_task> on 2024-03-27 18:49:47.272206+00:00
[2024-03-27 18:51:11,231] {standard_task_runner.py:52} INFO - Started process 495 to run task
[2024-03-27 18:51:11,236] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'bigdag', 'clean_task', 'a9b5d366-a1b9-4827-8e6a-0baba9a90cdc', '--job-id', '1378', '--raw', '--subdir', 'DAGS_FOLDER/dags.py', '--cfg-path', '/tmp/tmp9yx8xm39', '--error-file', '/tmp/tmp1g436vqs']
[2024-03-27 18:51:11,238] {standard_task_runner.py:80} INFO - Job 1378: Subtask clean_task
[2024-03-27 18:51:11,310] {task_command.py:369} INFO - Running <TaskInstance: bigdag.clean_task a9b5d366-a1b9-4827-8e6a-0baba9a90cdc [running]> on host 9b878320bbfc
[2024-03-27 18:51:11,382] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=bigdag
AIRFLOW_CTX_TASK_ID=clean_task
AIRFLOW_CTX_EXECUTION_DATE=2024-03-27T18:49:47.272206+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=a9b5d366-a1b9-4827-8e6a-0baba9a90cdc
[2024-03-27 18:51:11,386] {python.py:173} INFO - Done. Returned value was: None
[2024-03-27 18:51:11,397] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=bigdag, task_id=clean_task, execution_date=20240327T184947, start_date=20240327T185111, end_date=20240327T185111
[2024-03-27 18:51:11,418] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-03-27 18:51:11,443] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
